templates/graphdb/configmap-properties.yaml

```diff
@@ -20,6 +20,42 @@ data:
     {{- if eq (int .Values.replicas) 1}}
     graphdb.external-url={{ include "graphdb.external-url" . }}
     {{- end }}
+    {{- with .Values.cluster.config.tls }}
```

Comment:
You need to configure the proxies as well, otherwise they won't be able to grpc with the nodes.